lib/render/game.php

```diff
@@ -41,9 +45,38 @@ function gameAvatar(
         iconSize: $iconSize,
         iconClass: $iconClass,
         context: $context,
+        sanitize: $labelIsHtml === false,
```

Comment:
The point of sanitization is to escape HTML. It seems like this effectively circumvents that protection. Did you encounter a case where this was problematic? The `~Category~` tags shouldn't be affected by the sanitization process.

Reply:
The actual `$label` that gets passed in `gameAvatar(...)`'s call to `avatar(...)` is the already-rendered game title (e.g `<span>~Category~</span> Lorem Ipsum`), so sanitization can't happen in this case. But indeed, suspending it every time there's HTML doesn't make sense either. A simple solution is to just sanitize the title _before_ generating the HTML.